include/tfm/sdfw_psa_ipc_service.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+/*
+ * This header contains symbols that are used by both the SDFW client
+ * and the SDFW service.
+ */
+
+/* Number of uint32_t words in the data sent over ipc_service */
+#define SDFW_PSA_IPC_DATA_LEN 6
+
+/* Usually provided by manifest/sid.h */
```

Comment:
This isn't a very informative comment, sid.h even includes this file here?

Reply:
fixed